<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>UserWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>myfile/images/user.jpg</normaloff>myfile/images/user.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 230, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>myfile/images/user.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="zzz">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwdLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="userNameLEd">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>工程师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>操作员</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="pwdLEd">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/Mabe/arrow-right-circle.jpg</normaloff>myfile/images/Mabe/arrow-right-circle.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>46</width>
     <height>46</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/Mabe/arrow-left-circle.jpg</normaloff>myfile/images/Mabe/arrow-left-circle.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>46</width>
     <height>46</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
